plugins/hydra_optuna_sweeper/example/multi-objective-conf/config.yaml

```diff
@@ -0,0 +1,28 @@
+defaults:
+  - override hydra/sweeper: optuna
```

Comment:
just confirming that we do not plan to backport this to 1.0 correct?

Reply:
good point. If this plugin is targeting Hydra 1.1 setup.py should reflect that.

Reply:
Thank you for your question. This plugin will use the Hydra 1.1 feature as suggested in https://github.com/facebookresearch/hydra/pull/1294#discussion_r557767722, but it still works with Hydra 1.0 for now.

I think we can backport this change to 1.0, but do we have any difficulties?

Reply:
For now we will not have much difficulties but the above line ('override hydra/sweeper: optuna') will not work in 1.0.
other features we are likely to use in next iterations will make actual use of Hydra 1.1 capabilities and will not be portable.
I suggest that we limit the plugin to 1.1.
maintaining two versions of the plugin at this early stage is not worth it in my opinion. People that wants to use it can use Hydra 1.1 dev release and provide some additional testing for it.

Reply:
Thank you for your explanation. I was not aware of the use of Hydra 1.1 features.

> maintaining two versions of the plugin at this early stage is not worth it in my opinion.

I agree. The development resource is limited, and it sounds reasonable to me.

> People that wants to use it can use Hydra 1.1 dev release and provide some additional testing for it.

Ah, you already released [`v1.1.0.dev3`](https://pypi.org/project/hydra-core/1.1.0.dev3/) at PyPI. Users can easily use it with `pip install --pre hydra`.

Reply:
> We are not using any yet. but I mentioned that we can configure the sampler with Hydra.

Oh, I misunderstood. `override hydra/sweeper: optuna` is just used in the example config.

> Going with 1.1 does hurt the visibility of the plugin in the short term, but will also allow us to focus on making it better, leveraging 1.1 features.

Agreed. I'd like to focus on the improvement with the Hydra 1.1 features as you mentioned. I'll start from the [recursive defaults](https://github.com/facebookresearch/hydra/pull/1294#discussion_r568890895).